packages/components/src/InputEmail/InputEmail.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { cleanup, fireEvent, render, waitFor } from "@testing-library/react";
+import { InputEmail } from ".";
+
+afterEach(cleanup);
+
```

Comment:
We need to make sure while the user is typing for the first time, we shouldn't show a validation error. Can we have a test to verify that we don't show a validation error after the user has typed an invalid email but the input has not blured?

Reply:
Addressed here: https://github.com/GetJobber/atlantis/pull/610/commits/4d030df5b37aecd6e129afa47066a4e1b07ce95e